doc/pages/faq.asciidoc

```diff
@@ -1,44 +1,48 @@
 = FAQ
 
-== How to pronounce the name of the project and what does it mean ?
+== The project
+
+=== How to pronounce the name of the project and what does it mean ?
 
 The name of the project is pronounced "Kak-oon", and is a word taken from a
 New Caledonian dialect based on French. It means a hard blow, usually a punch,
-but generally refers to a blow into which all of one's strength went.
+but generally refers to a strike into which all of one's strength went.
+
+=== Is there going to be a Windows port of Kakoune ?
 
-== Is there going to be a Windows port of Kakoune ?
+As many features provided by UNIX systems are missing or less efficient on
```

Comment:
```suggestion
No, as many features provided by UNIX systems are missing or less efficient on
```

Reply:
The FAQ doesn't claim there will not be a Windows port in the future, only that the incentive is low.

Answering the question with a "no" would be less future-proof that what we have currently, so I don't think we should apply this hunk.